suzieq/engines/pandas/interfaces.py

```diff
@@ -35,16 +55,45 @@ def get(self, **kwargs):
         if df.empty:
             return df
 
-        if state:
-            if state.startswith('!'):
-                df = df.query(f'state != "{state[1:]}"')
-            else:
-                df = df.query(f'state=="{state}"')
+        if portmode or 'portmode' in columns or '*' in columns:
+            for x in ['ipAddressList', 'ip6AddressList']:
+                if x in columns or '*' in columns:
+                    continue
+                drop_cols.append(x)
+            df = self._add_portmode(df)
+
+        if vlan or "vlanList" in columns or '*' in columns:
+            if 'portmode' not in columns and '*' not in columns:
+                for x in ['ipAddressList', 'ip6AddressList']:
+                    if x in columns or '*' in columns:
+                        continue
```

Comment:
At line 68, the `'*' in columns` is useless since you can enter there only if `'*' not in columns`

Reply:
Good catch, will fix